### ISSUE TITLE
Floor or Ceiling operator using functional API

### ISSUE THREAD

**Issue by edwin-19** [NVIDIA/DALI#2630]
Hi, I've got a question is there any operator that resembles ceiling or floor?

 I used a round about way to round off but in the end is closer to 'round' instead of ceiling or floor
 ```python
fn.cast(fn.cast(ratio * size, dtype=types.INT64), dtype=types.FLOAT) # array(818., dtype=float32), closer to the round(n, 2) function
 ```
 Btw i am using the functional api to build my graph.
 
Or is it not possible with the current operators?

**Comment by klecki** [comment]
Hi,
indeed the `Cast` op is doing rounding.
To solve the problem I suggest adding 0.5 for ceil and subtracting 0.5 for floor. It can work with scalars:
```
def ceil(x):
	return fn.cast(fn.cast(x + 0.5, dtype=types.INT64), dtype=types.FLOAT)

def floor(x):
	return fn.cast(fn.cast(x - 0.5, dtype=types.INT64), dtype=types.FLOAT)
```

We have some more common mathematical functions in our TODO list.

**Comment by edwin-19** [comment]
Hi thanks this solved the problem.

Oo looking forward to the common mathematical functions

**Comment by JanuszL** [comment]
If you have some spare time you can add them on your own in a similar fashion as in https://github.com/NVIDIA/DALI/pull/2555 PR.

**Comment by klecki** [comment]
Hi, @edwin-19, I just merged #2853 that adds more mathematical ops to DALI, `floor` and `ceil` included.
It should appear in next nightly build and be a part of DALI 1.2 release.
